docs/developers/Contributing_Guidelines.md

```diff
@@ -0,0 +1,275 @@
+# Contributing Guidelines
```

Comment:
This file is quite decent, several places that would benefit from human review, but no massive problems or misleading info